script.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import pprint
+import config
+
+def main():
+    token = generateToken(config.client_id, config.client_secret)
+    if token:
+        response = requestVehicles(token)
+        printInfo(response)
+    return 0
+
+def generateToken(client_id, client_secret):
+    tokenResponse = requests.post("https://api.oregonstate.edu/oauth2/token", data={
+        "client_id": client_id,
+        "client_secret": client_secret,
+        "grant_type": "client_credentials"
+    })
+    if(tokenResponse.status_code == 200):
+        return tokenResponse.json()['access_token']
+    else:
+        print("Invalid credentials")
+        return None
+
+def requestVehicles(token):
+    parameters = {
+        "Authorization": f"Bearer {token}"
+    }
+    return requests.get("https://api.oregonstate.edu/v1/beaverbus/vehicles", headers=parameters)
```

Comment:
Current, you only send a request to this endpoint which will always return all vehicles whenever we run the script. Can you figure a way to prompt your users to enter input and search vehicles based on that input? For example:

1. If enter `all`, list all vehicles.
2. If enter `1` return the info of the vehicle with ID is 1.
3. Return some errors message if the vehicle with the ID not found.